distribution/command-line/src/main/kotlin/org/codice/ckt/Command.kt

```diff
@@ -2,7 +2,7 @@ package org.codice.ckt
 
 import us.jimschubert.kopper.Parser
 
-const val distroDir = "/distribution/command-line/target/command-line-1.0-SNAPSHOT-bin"
+val samlDist = "${System.getProperty("user.dir")}/.."
```

Comment:
✏️ Not necessary for this PR, but we should be able to canonicalize this path and simplify.

Reply:
I'll add that to the ticket to fix the script